doc/source/_static/override.css

```diff
@@ -0,0 +1,3 @@
+code {
+    white-space: normal;
```

Comment:
What was the reasoning for adding this override?

Reply:
It doesn't brake long line of code if it doesn't have any EOL symbols in it which looks not good.